TreeMaker/python/maker.py

```diff
@@ -103,6 +106,27 @@ def __init__(self,parameters):
 
         self.readFiles = [(self.redir if val.startswith("/") else "")+val for val in self.readFiles]
 
+        # https://htcondor.readthedocs.io/en/latest/man-pages/condor_chirp.html?highlight=set_job_attr_delayed#chirp-commands
+        # https://github.com/cms-sw/cmssw/blob/master/FWCore/Services/plugins/CondorStatusUpdater.cc#L377-L410
+        try:
+            if os.environ["_CONDOR_CHIRP_CONFIG"]:
+                key = "ChirpTreeMakerReadFiles"
+                value = "\"" + ",".join(self.readFiles) + "\""
+                try:
+                    chirp_command = "condor_chirp set_job_attr_delayed " + key + " " + value
+                    proc = subprocess.Popen(chirp_command.split(), shell = False, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
+                    outs, errs = proc.communicate()
+                except Exception as e:
+                    proc.kill()
+                    print "An exception occurred: {}".format(e)
+                finally:
+                    if proc.returncode:
+                        print "WARNING::condor_chirp failed to set the attribute \'" + key + "\'" 

```

Comment:
I think this permissive approach is okay for now, but we might want to make it an exception at some point

Reply:
i would probably just put a TODO comment in the code, but you can open an issue if you prefer

Reply:
A comment has been added to the code.